torch/csrc/jit/passes/loop_unrolling.cpp

```diff
@@ -162,10 +162,11 @@ void replaceLoopCounter(Node* loop) {
   body->insertOutput(1, result);
 }
 
-void unroll(Node* loop) {
+void unroll(Node* loop, bool constant_only) {
   Graph* graph = loop->owningGraph();
   Block* body = loop->blocks().at(0);
-  if (!isSmallBlock(body))
+
+  if (!isSmallBlock(body) && !constant_only)
```

Comment:
[Readability issue] This logic is a bit confusing. If I only read the implementation of `unroll()`, it seems to unroll both small-block loops and constant loops (regardless of sizes). But then the name `constant_only` is confusing.

But If I read the invocation contact of unroll, it is clear that when `constant_only` is true, we only unroll constant loops.
```
     if (constant_only &&
           node->inputs().at(0)->node()->kind() != prim::Constant) {
         continue;
       }
      unroll(node, constant_only);
```

For better readability, we either push the checking for `prim::Constant` into `unroll()`; or move the checking logic out of `unroll()` and remove `constant_only` parameter to `unroll()` so that `unroll()` just unroll a loop (all the unrolling decision-making is done before calling `unroll()`.